src/helix/mod.rs

```diff
@@ -355,8 +355,8 @@
 //!
 //! | Endpoint | Helper | Module |
 //! |---|---|---|
-//! | [Get Users](https://dev.twitch.tv/docs/api/reference#get-users) | [`HelixClient::get_user_chat_color`], [`HelixClient::get_user_from_id`], [`HelixClient::get_users_chat_colors`], [`HelixClient::get_user_from_login`], [`HelixClient::get_users_from_ids`] | [`users::get_users`] |
-//! | [Update User](https://dev.twitch.tv/docs/api/reference#update-user) | [`HelixClient::update_user_chat_color`] | - |
+//! | [Get Users](https://dev.twitch.tv/docs/api/reference#get-users) | [`HelixClient::get_user_from_id`], [`HelixClient::get_user_from_login`], [`HelixClient::get_users_chat_colors`], [`HelixClient::get_users_from_ids`] | [`users::get_users`] |
```

Comment:
`get_users_chat_colors` shouldn't be here, to fix this I think we'd have to apply some kind of distance function maybe...